builder/googlecompute/driver_gce.go

```diff
@@ -126,11 +128,11 @@ func NewClientOptionGoogle(account *ServiceAccount, vaultOauth string, impersona
 		// 4. On Google Compute Engine and Google App Engine Managed VMs, it fetches
 		//    credentials from the metadata server.
 		//    (In this final case any provided scopes are ignored.)
-		//
-		//    Note: (4) is not usable with OSLogin on Google Compute Engine (GCE).
-		//    The GCE service account is derived separately and used instead.
 	}
 
+	// Insert UserAgents
+	opts = append(opts, option.WithUserAgent(useragent.String(version.PluginVersion.FormattedVersion())))
```

Comment:
Should this be more precise, something like "packer-plugin-googlecompute-v1.2.3" ?

Reply:
Instead of doing 
`service.UserAgent = useragent.String(version.PluginVersion.FormattedVersion())` 3 times inside `NewClientOptionGoogle()` for each service, I inserted the useragent much earlier in the auth options. Was the string incorrect before?

Reply:
Also, that UA is propagated correctly to the post-processor clients as well. 😄

Reply:
No I think this is correct, while reading it I just thought we could be a bit more precise.